data/src/main/java/com/linku/data/implementation/repository/UserRepositoryImpl.kt

```
@@ -92,102 +47,46 @@ class UserRepositoryImpl @Inject constructor(
     override suspend fun getNickname(userId: Long): String? {
         return try {
-            val dto = serverApi.withAuth(authPreference) {
-                getUserInfo(/*userId*/)
-            }
-            val nick = dto.nickName
-            Log.d(TAG, "닉네임=$nick")
-            nick?.takeIf { it.isNotBlank() }
-        } catch (e: ApiError) {
-            Log.e(TAG, "닉네임 가져오기 실패: ${e.message}")
+            serverApi.getUserInfo().result.nickName?.takeIf { it.isNotBlank() }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)